### PR TITLE
Stepping down as editor

### PR DESCRIPTION
**Pull request by esc**

### CODE REVIEW

**Comment by esc** [comment]
Be seeing you!

**Comment by esc** [comment]
I also have removed myself from the organization, going through some cleanup right now.

**Comment by GaelVaroquaux** [comment]
We'll miss you @esc ! Thanks for being on board. You really made an impact.

File: AUTHORS.rst [hunk 1]
```diff
@@ -42,6 +40,8 @@ Listed by alphabetical order.
 
 - Zbigniew Jędrzejewski-Szmek
 
+- Valentin Haenel
```
Comment (GaelVaroquaux) [inline AUTHORS.rst]: You should add that you were editor for years. It's important!

Comment (GaelVaroquaux) [inline AUTHORS.rst]: Addressed!

**Comment by esc** [comment]
ty

**Pull request merged**